target/linux/ramips/dts/U25AWF.dts

```diff
@@ -0,0 +1,108 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/{
+	compatible="u25awf","ralink,mt7620n-soc";
+	model="KimaxU-25AWF";
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+		reset {
+			label = "reset";
+			gpios = <&gpio2 3 GPIO_ACTIVE_HIGH>;
```

Comment:
This is the incorrect GPIO_ACTIVE level, you had '1' in the original patch which is `GPIO_ACTIVE_LOW`.